packages/eslint-plugin/src/rules/jsx-pragma/__tests__/rule.test.ts

```diff
@@ -64,21 +74,30 @@ tester.run('jsx-pragma', jsxPragmaRule, {
     },
     {
       name: 'should be valid if runtime = classic and classic jsx pragma is used with css function call',
-      code: `
+      code: outdent`
         /** @jsx jsx */
         import { css, jsx } from '@compiled/react';
         <div css={css({ display: 'block' })} />
       `,
       options: [{ runtime: 'classic' }],
     },
+    {
+      name: 'should be valid if runtime = classic and classic jsx pragma is used with css function call found in importSources',
+      code: outdent`
+        /** @jsx jsx */
+        import { css, jsx } from '@atlaskit/css';
```

Comment:
This is the bulk of the change _we need_, but I go on to fix the automated variants as well, works as far as I'm aware..